.github/workflows/auto-release.yml

```diff
@@ -83,24 +95,32 @@ jobs:
           source_path: ./
 
       - name: commit SBOM file
+        env:
+          VERSION_NUMBER: ${{ github.event.inputs.version_number }}
         working-directory: ./local_kernel
         run: |
           git add .
           git commit -m '[AUTO][RELEASE]: Update SBOM'
-          git push -u origin ${{ github.event.inputs.version_number }}
+          git push -u origin "$VERSION_NUMBER"
           echo "COMMIT_SHA_2=$(git rev-parse HEAD)" >> $GITHUB_ENV
 
       - name: Release
+        env:
+          VERSION_NUMBER: ${{ github.event.inputs.version_number }}
+          MAIN_BR_VERSION_NUMBER: ${{ github.event.inputs.main_br_version }}
+          COMMIT_SHA_2: ${{ env.COMMIT_SHA_2 }}
+          REPO_OWNER: ${{ github.repository_owner }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Same as above - the `GITHUB_TOKEN` variable looks unused.

Reply:
Refer to [result](https://github.com/ActoryOu/FreeRTOS-Kernel/actions/runs/11633424178/job/32398542219), this is used in py script.

> Traceback (most recent call last):
>   File "/home/runner/work/FreeRTOS-Kernel/FreeRTOS-Kernel/./tools/.github/scripts/release.py", line 625, in <module>
>     main()
>   File "/home/runner/work/FreeRTOS-Kernel/FreeRTOS-Kernel/./tools/.github/scripts/release.py", line 576, in main
>     assert 'GITHUB_TOKEN' in os.environ, 'Set env{GITHUB_TOKEN} to an authorized git PAT'
>            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> AssertionError: Set env{GITHUB_TOKEN} to an authorized git PAT
> Error: Process completed with exit code 1.